man/systemd.exec.xml

```diff
@@ -89,6 +89,15 @@
 
     <variablelist class='unit-directives'>
 
+      <varlistentry>
+        <term><varname>ExecSearchPaths=</varname></term>
+
+        <listitem><para>Takes a list of absolute paths in the file system relative to which the executable used by
```

Comment:
nit: I'd prefer if the wording was consistent with how we describe other properties. i.e. `Takes a space-separated list of...`

Reply:
This should also say that it overrides PATH instead of takes precedence.